command/job_scale.go

```diff
@@ -144,7 +153,19 @@ func (j *JobScaleCommand) Run(args []string) int {
 
 	// Perform the scaling action.
 	w := &api.WriteOptions{Namespace: namespace}
-	resp, _, err := client.Jobs().Scale(jobID, groupString, &count, msg, false, nil, w)
+	req := &api.ScalingRequest{
+		Count: pointer.Of(int64(count)),
+		Target: map[string]string{
+			"Job":   jobID,
+			"Group": groupString,
+		},
+		Message:        msg,
+		PolicyOverride: false,
+		EnforceIndex:   checkIndex > 0,
+		JobModifyIndex: checkIndex,
```

Comment:
why the separate `EnforceIndex` bool, when it's derived anyway?  `job_endpoint.go` below has `args.JobModifyIndex != 0` already -- am I missing what we gain by having the extra bool?

Reply:
That's a good point... I was being consistent with the job register API where having both flags does matter. The Raft log we write is the `JobRegister` log entry here anyways. Will fix.